plugins/addon/controller.go

```diff
@@ -54,6 +55,7 @@ var _ reconcile.Reconciler = &{{ .Resource.Kind }}Reconciler{}
 type {{ .Resource.Kind }}Reconciler struct {
 	client.Client
 	Log logr.Logger
+	Scheme *runtime.Scheme
```

Comment:
Plugins may not need an Scheme, why should you enforce them to have that field?

Reply:
Please, see the above explanation which described why it is required.

Reply:
Same as above, mark as resolved please.